include/shell.h

```diff
@@ -153,25 +154,9 @@ class DOS_Shell : public Program {
 	bool call = false;
 };
 
-/* Object to manage lines in the autoexec.bat The lines get removed from
- * the file if the object gets destroyed. The environment is updated
- * as well if the line set a a variable */
-class AutoexecObject{
-private:
-	bool installed = false;
-	std::string buf = {};
-
-public:
-	AutoexecObject() = default;
-	AutoexecObject(const std::string& line);
-	~AutoexecObject();
-
-	void Install(const std::string& in);
-	void InstallBefore(const std::string& in);
-
-private:
-	void CreateAutoexec();
-};
+// XXX
```

Comment:
Okay, you've moved this from here, including the existing comments. Then my remarks about the bad comments are directed at the original devs, not you 😄 

Still, as you must have spent considerable amount of time understanding and improving this, could you document this class as per my previous comments? It's not exactly trivial how these "objects" are used.

Reply:
Objects are now gone.